frontend/app/src/components/accounts/ManualBalancesList.vue

```diff
@@ -194,11 +173,22 @@ export default class ManualBalancesList extends Vue {
   exchangeRate!: (currency: string) => number;
 
   get visibleBalances(): ManualBalance[] {
-    return this.manualBalances.filter(balances => {
-      return this.onlyTags.every(tag => balances.tags.includes(tag));
+    if (this.onlyTags.length === 0) {
+      return this.manualBalances;
+    }
+
+    return this.manualBalances.filter(balance => {
+      if (balance.tags) {
+        return this.onlyTags.every(tag => balance.tags.includes(tag));
+      }
     });
   }
 
+  editBalance(balance: ManualBalance) {
+    this.$emit('editBalance', balance);
+    return balance;
```

Comment:
is this really needed here?

Reply:
Open to suggestions on a more elegant way to do it.

Reply:
What I am really confused about is why we need to return the balances at the click handler. I am under the impression that the `@click` is just a void function.

Reply:
I guess we don't. I think initially I had tried to do it with the Emit() decorator (which seems to need to return something) and this was just left over. Fixed.

Reply:
You don't really need to return something with emit unless you want to transform the data :)

Reply:
Should be fixed in latest commit